docs/content/reference/dynamic-configuration/file.toml

```diff
@@ -463,7 +463,17 @@
       [tls.stores.Store0.defaultCertificate]
         certFile = "foobar"
         keyFile = "foobar"
+      [tls.stores.Store0.defaultGeneratedCert]
+        resolver = "foobar"
+        [tls.stores.Store0.defaultGeneratedCert.domain]
+          main = "foobar"
+          sans = ["foobar", "foobar"]
```

Comment:
Should these also have the `example.org` / `foo.example.org` / `bar.example.org` nomenclature used in the docs?

Reply:
not the reference files